airbyte/_executors/python.py

```diff
@@ -106,6 +148,14 @@ def install(self) -> None:
 
         After installation, the installed version will be stored in self.reported_version.
         """
+        package_name = (
+            self.metadata.pypi_package_name
+            if self.metadata and self.metadata.pypi_package_name
+            else f"airbyte-{self.name}"
+        )
+        requires_python = _get_pypi_python_requirements_cached(package_name)
+        self._check_python_version_compatibility(package_name, requires_python)
```

Comment:
Let's make it explicit we are no-op-ing the return value on purpose.
I'll apply this and you can pull the latest.

```suggestion
         _ = self._check_python_version_compatibility(package_name, requires_python)
```